client/components/boards/boardHeader.js

```diff
@@ -15,6 +15,19 @@ Template.boardMenuPopup.events({
   }),
 });
 
+Template.boardMenuPopup.helpers({
+  exportUrl() {
+    const boardId = Session.get('currentBoard');
+    const userId = Meteor.userId();
+    const loginToken = Accounts._storedLoginToken();
+    return Meteor.absoluteUrl(`api/b/${boardId}/${userId}/${loginToken}`);
```

Comment:
- using only the token, and as a parameter, is great. That way, we can use the same route for api access, just with code looking for token in parameter then in header. I like that :heart: 
- I'm not a fan of using an "export" method name, it seems non-RESTy, we just want a given board: `api/boards/${boardId}?authToken=${loginToken}`
- the format parameter seems overkill for the time being. I suggest we add it later if needed (and keep current JSON as the default). But when we add it, we might use it to specify versioning as well (like `?format=wekan-board-1.0.0.json`)